src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
+@import './vars';
+@import './mixins';
+@import './keyframes';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  @include stopwatch-flex-property;
+}
+
+.stopwatch {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  color: #000;
+
+  @include stopwatch-flex-property;
+  @include stopwatch-border-style(1vmin, dotted, 50%);
+
+  &__hand {
+    bottom: 50%;
+    transform-origin: bottom;
+    position: absolute;
+
+    @include stopwatch-flex-property;
+
+    &--min {
+      @include stopwatch-elements-styles(
+        $stopwatch-min-width,
+        $stopwatch-min-height,
+        $stopwatch-colors-min
+      );
+
+      animation: watch-animation-rotate 3600s infinite steps(60);
+    }
+
+    &--second {
+      @include stopwatch-elements-styles(
+        $stopwatch-second-width,
+        $stopwatch-second-height,
+        $stopwatch-colors-second
+      );
+
+      animation: watch-animation-rotate 60s infinite linear;
+    }
+
+    /* Правила для прискорення годинника */
+
+    @each $hand, $second in $speed-up {
+      &--#{$hand}--speed-up {
+        animation-duration: $second;
+      }
+    }
```

Comment:
The usage of the SCSS `@each` directive is correct here. It makes your code more DRY and maintainable. However, the naming of the variable `$second` is a bit confusing. It would be better to name it something like `$duration` to make it more clear what it represents.